### ISSUE TITLE
Question: unclear what size of a time series to use for the multi-input solar model

### ISSUE THREAD

**Issue by rbavery** [allenai/satlas#30]
the paper describes that 8 input scenes are used

> For multi-image SATLASNET, we provide as input a time series of eight Sentinel-2 images for low-resolution mode

however this document describes using 3 scenes for the multi image models in the custom inference example: https://github.com/allenai/satlas/blob/main/Normalization.md#sentinel-2-images

can the model inference effectively on any sized time series? Or is it best to only use a time series of 8 input scenes since the model was originally trained with that time series length?

**Comment by favyen2** [comment]
The model is pre-trained with 8 input images (https://github.com/allenai/satlas/blob/main/configs/sentinel2/mi_mb_base.txt).

It is then fine-tuned on hand-labeled solar farm dataset, where it inputs 4 images (https://github.com/allenai/satlas/blob/main/configs/satlas_explorer_solar_farm.txt).

So inference should be done on 4 images.

I have updated the documentation to clarify where to find the number of images needed for different models.
